.github/workflows/deploy-webui.yml

```diff
@@ -0,0 +1,61 @@
+name: Deploy webui to Github Pages
+
+on:
+  # Runs on pushes targeting the webui branch
+  push:
+    branches: ['webui']
```

Comment:
update to push and PR for the master branch?

Reply:
Done.

Reply:
still looks unchanged?